dev/io.openliberty.microprofile.openapi.4.0.internal/src/io/openliberty/microprofile/openapi40/internal/impl/OpenAPI31ModelOperations.java

```diff
@@ -33,7 +30,6 @@
 import io.smallrye.openapi.runtime.io.IOContext;
 import io.smallrye.openapi.runtime.io.JsonIO;
 
-@Component(configurationPolicy = ConfigurationPolicy.IGNORE, service = OpenAPIModelOperations.class)
 public class OpenAPI31ModelOperations extends OpenAPIModelOperationsImpl {
```

Comment:
This is the main reason for refactoring this code into a common project for 4.0 and 4.1. We need a slightly different implementation of this service in 4.1.

There are two new classes in the other bundles which extend this class and declare a component.